### PR TITLE
feat: add foxglove websocket 1.3.0

### PR DESCRIPTION
**Pull request by wep21**
- as title
- need skip url stability check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (foxglove_websocket) have been updated in this PR. Please review the changes.

**Comment by wep21** [comment]
@fmeum I've drop windows ci.

**Review by fmeum — APPROVED** [review]

File: modules/foxglove_websocket/1.3.0/overlay/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,16 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "foxglove_websocket",
+    srcs = glob(["cpp/foxglove-websocket/src/*.cpp"]),
+    hdrs = glob(["cpp/foxglove-websocket/include/foxglove/websocket/*.hpp"]),
+    copts = ["-std=c++17"],
```
Comment (fmeum) [inline modules/foxglove_websocket/1.3.0/overlay/BUILD.bazel:9]: Could this be moved to the build flags? It wouldn't necessarily be safe to mix standards in a single build.

**Review by fmeum — CHANGES_REQUESTED** [review]

File: modules/foxglove_websocket/1.3.0/overlay/BUILD.bazel (hunk 1 above)
Comment (wep21) [reply modules/foxglove_websocket/1.3.0/overlay/BUILD.bazel]: I've moved it to presubmit.yml

**Review by fmeum — APPROVED** [review]

**Pull request merged**